docs/section5/Rule5-26.md

```diff
@@ -18,79 +18,77 @@
 
-      - Else if window-wall-ratio is between 20% to 30%, get baseline vertical fenestration construction SHGC requirement: `if ( wwr_semiheated > 20.0% ) AND ( wwr_semiheated <= 30.0% ): target_shgc_semiheated = data_lookup(table_G3_4, climate_zone, "SEMIHEATED", "VERTICAL GLAZING", "20.1%-30.0%", "ASSEMBLY MAX. SHGC")`
+    - Else, window-wall-ratio is between 30% to 40%, get baseline vertical fenestration construction SHGC requirement: `if ( wwr_semiheated > 30.0% ) AND ( wwr_semiheated <= 40.0% ): target_shgc_semiheated = data_lookup(table_G3_4, climate_zone, "SEMIHEATED", "VERTICAL GLAZING", "30.1%-40.0%", "ASSEMBLY MAX. SHGC")`
 
-      - Else, window-wall-ratio is between 30% to 40%, get baseline vertical fenestration construction SHGC requirement: `if ( wwr_semiheated > 30.0% ) AND ( wwr_semiheated <= 40.0% ): target_shgc_semiheated = data_lookup(table_G3_4, climate_zone, "SEMIHEATED", "VERTICAL GLAZING", "30.1%-40.0%", "ASSEMBLY MAX. SHGC")`
+- For each zone in B_RMR: `for zone_b in B_RMR...zones:`
 
-  - For each zone in building: `for zone_b in building_b...zones:`
+  - For each surface in zone: `for surface_b in zone_b.surfaces:`
 
-    - For each surface in zone: `for surface_b in zone_b.surfaces:`
+    - Check if surface is above-grade wall with subsurface and is regulated: `if ( get_opaque_surface_type(surface_b) == "ABOVE-GRADE WALL" ) AND ( surface_b.subsurfaces ) AND ( scc_dictionary_b[surface_b.id] != "UNREGULATED" ):`
 
-      - Check if surface is above-grade wall with subsurface and is regulated: `if ( get_opaque_surface_type(surface_b) == "ABOVE-GRADE WALL" ) AND ( surface_b.subsurfaces ) AND ( scc_dictionary_b[surface_b.id] != "UNREGULATED" ):`
+      - For each subsurface in wall: `for subsurface_b in surface_b.subsurfaces:`
 
-        - For each subsurface in wall: `for subsurface_b in surface_b.subsurfaces:`
+        - Check if subsurface is door and glazed area is more than 50% of the total door area, or subsurface is not door: `if (( subsurface_b.classification == "DOOR" ) AND ( subsurface_b.glazed_area > subsurface_b.opaque_area )) OR ( subsurface_b.classification != "DOOR" ):`
 
-          - Check if subsurface is door and glazed area is more than 50% of the total door area, or subsurface is not door: `if (( subsurface_b.classification == "DOOR" ) AND ( subsurface_b.glazed_area > subsurface_b.opaque_area )) OR ( subsurface_b.classification != "DOOR" ):`
+          **Rule Assertion - Component:**
 
-            **Rule Assertion - Component:**
+          - Case 1: For each subsurface that is vertical fenestration, if wall is exterior mixed type and the baseline requirements for residential and non-residential type SHGC for different window-wall-ratio are different: `if manual_review_flag AND ( scc_dictionary_b[surface_b] == "EXTERIOR MIXED" ): UNDETERMINED`
 
```

Comment:
Comment from @mkarpman 
One improvement that may be made to both 5-26 and 5-37 to reduce occurance of "undetermined" outcome is to check whether "mixed" fenestration would affect the WWR bracket for either residential or non-residential.  Currently, the indeterminate outcome is produced if the U-factor in Tables G3.4 differs for residential vs non-residential and WWR bracket. However, if area or "mixed" fenestration is small enough to not result in a change of WWR bracket for either residential or non-residential, the rule can produce PASS or FAIL outcome instead of UNDETERMINED.

Reply:
as discussed, since the function has already been implemented, we will leave it as is and add a comment to the RDS to clarify that this workaround can be implemented in the future

Reply:
Note added in R-26 and get_rmi_scc_window_wall_ratios().